microservice/src/test/java/nearsoft/skytouch/microservice/repository/ProductRepositoryTest.java

```diff
@@ -0,0 +1,84 @@
+package nearsoft.skytouch.microservice.repository;
+
+import nearsoft.skytouch.common.model.Product;
+import nearsoft.skytouch.microservice.repository.impl.ProductRepositoryImpl;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import javax.persistence.EntityManager;
+import javax.persistence.ParameterMode;
+import javax.persistence.PersistenceContext;
+import javax.persistence.StoredProcedureQuery;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class ProductRepositoryTest {
```

Comment:
I strongly recommend to change the test to validate the execution of the actual store procedures

Reply:
Ok i will test it with a testing database thanks